frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -61,33 +86,144 @@ export default {
     }
   },
   validations: {
-    form: {
-      member: {
-        required
-      }
-    },
     // validation groups by route name for steps
     steps: {
       Member: [
         'form.member'
       ]
     }
   },
+  computed: {
+    ...mapState([
+      'currentGroupId',
+      'loggedIn'
+    ]),
+    ...mapGetters([
+      'currentGroupState',
+      'groupSettings',
+      'groupMembersCount',
+      'groupShouldPropose'
+    ])
+  },
   methods: {
-    submit (form) {
-      console.log(
-        'TODO: Logic to Propose Mincome.',
-        'mincome:', this.form.member,
-        'reason:', form.reason
-      )
+    async addInvitee (index) {
+      const searchUser = this.form.invitees[index]
+
+      if (searchUser === this.$store.state.loggedIn.username) {
+        return this.setInviteError(index, this.L('You can\'t invite yourself'))
+      }
+
+      if (this.currentGroupState.profiles[searchUser]) {
+        return this.setInviteError(index,
+          this.L('{searchUser} is already on the group', { searchUser })
```

Comment:
Can you replace all instances of `this.L` with `L`? It's shorter and looks cleaner that way

Reply:
🤔 in the previous PR you asked me to replace it with L function and now you are saying to use this.L again... what's the final decision?

Reply:
nevermind, i'm still waking up 🤦‍♀